### DISCUSSION TITLE
为什么32g内存16g显存batchsize=1跑不起来(stable diffusion )

### DISCUSSION THREAD

**Discussion by yufengyao-lingoace** [hpcaitech/ColossalAI#2341] [Community | Q&A]
stable diffusion 为什么32g内存16g显存batchsize=1跑不起来

**Comment by feifeibear** [comment]
你用的什么脚本？报什么错误

**Reply by yufengyao-lingoace** [reply]
你好就是用的colosssal最新的代码:https://github.com/hpcaitech/ColossalAI/tree/main/examples/images/diffusion.
有个问题，type object 'ChunkManager' has no attribute 'search_chunk_size，然后换了colossal0.1.10,不报这个错了。
Epoch 0:   0%|
Summoning checkpoint.
内存一直飙升，最后32g内存和15g虚拟内存都用完了，
最后显示Killed

**Reply by yufengyao-lingoace** [reply]
killed之前，显存也升到了最高值，15g多

**Reply by yufengyao-lingoace** [reply]
运行的这个脚本：python main.py --logdir /tmp/ -t -b configs/Teyvat/train_colossalai_teyvat.yaml

**Reply by yufengyao-lingoace** [reply]
已经解决了，还是得安装合适的版本才行
